build_scripts/OSX/MacOS_X_InstallGuide.md

```diff
@@ -104,11 +119,11 @@ In QtCreator Projects -> Build -> Build Settings -> Build Steps -> Add Additiona
 
 ## Set your Mac OS SDK version
 
-Edit RetroShare.pro  
+Edit RetroShare.pro and set your installed sdk version example for 11.1 -> rs_macos11.1
 
     CONFIG += c++14 rs_macos11.1
 
-and then retroshare.pri
+Edit retroshare.pri
```

Comment:
Why does the user need to edit retroshare.pri? There should be a section for MacOS that does the work, right?

Reply:
you need look latest changes
**Edit RetroShare.pro and set your installed sdk version example for 11.1 -> rs_macos11.1**
https://github.com/RetroShare/RetroShare/pull/2797/commits/31552b111914446d040faceb15c51d80c7488218

Reply:
![image](https://github.com/RetroShare/RetroShare/assets/9952056/5e5cb823-2973-4f2a-b3be-17793470646c)

Reply:
> Why does the user need to edit retroshare.pri? There should be a section for MacOS that does the work, right?

i has a newer version for edit
`Edit retroshare.pri and set your installed sdk version example for 11.1 -> rs_macos11.1 ( line 129:)`